packages/docs/src/page-configs/ui-elements/stepper/examples/Custom.vue

```diff
@@ -0,0 +1,118 @@
+<template>
```

Comment:
It's still very colorful. Maybe we can show customization in different ways?

<details>
<summary>Example</summary>

![image](https://user-images.githubusercontent.com/29167241/219009833-27a28264-33be-4c76-bd99-e15899f99c97.png)
The author is © [Nemo Fazakerley](https://dribbble.com/azc2012).
</details>

Reply:
I really like this idea.